evalml/utils/woodwork_utils.py

```diff
@@ -118,42 +117,47 @@ def _schema_is_equal(first, other):
     return logical and semantic
 
 
-def downcast_nullable_types(X, ignore_null_cols=True):
-    """Downcasts IntegerNullable, BooleanNullable types to Double and Boolean respectively.
+def downcast_nullable_types(data, ignore_null_cols=True):
+    """Downcasts IntegerNullable, BooleanNullable types to Double, Boolean in order to support certain estimators like ARIMA, CatBoost, and LightGBM.
 
     Args:
-        X (pd.DataFrame): Feature data.
+        data (pd.DataFrame, pd.Series): Feature data.
         ignore_null_cols (bool): Whether to ignore downcasting columns with null values or not. Defaults to True.
 
     Returns:
-        X: DataFrame initialized with logical type information where BooleanNullable are cast as Double.
+        data: DataFrame or Series initialized with logical type information where BooleanNullable are cast as Double.
     """
-    if X.ww.schema is None:
-        X.ww.init()
+    if data.ww.schema is None:
+        data.ww.init()
 
-    X_bool_nullable_cols = X.ww.select("BooleanNullable")
-    X_int_nullable_cols = X.ww.select(["IntegerNullable", "AgeNullable"])
+    if isinstance(data, pd.Series):
```

Comment:
if the series contains null values should we still downcast into Boolean or Double? I think we should follow what we have and use `ignore_null_cols` to differentiate.